src/package.ts

```diff
@@ -53,4 +53,8 @@ export function md5_file(file: string): Promise<string> {
             reject(error);
         }
     });
-}
\ No newline at end of file
+}
+
+export function validatePackageVersion(packageYmlVersion: string, releaseVersion: string): boolean {
+    return packageYmlVersion === releaseVersion;
```

Comment:
vielleicht sollte man auch varianten unterstützen.

z.B.
tag: `v1.0.0`
package.yml: `1.0.0`

Reply:
@gharlan @staabm meinung dazu?

Reply:
In anderen Projekten nutze ich bei den Tags teils die Notation mit `v...`. Bei den REDAXO-Addons scheint es mir bisher aber unüblich zu sein.
Daher könnte man mit deiner jetzigen Version starten und dann schauen, ob die Situation mit `v...`-Tags irgendwo mal auftritt.

Reply:
sehe ich auch so. ich habe auch mal fix FOR projekte durchgesehen und keine finden können, wo die tags mit den release versionen abweichen